runelite-api/src/main/java/net/runelite/api/VarPlayer.java

```diff
@@ -162,7 +162,9 @@
 	MUSIC_TRACKS_UNLOCKED_16(1009),
 	MUSIC_TRACKS_UNLOCKED_17(1338),
 	MUSIC_TRACKS_UNLOCKED_18(1681),
-	MUSIC_TRACKS_UNLOCKED_19(2065);
+	MUSIC_TRACKS_UNLOCKED_19(2065),
+
+	WINTERTODT_TIMER(1142);
```

Comment:
What is this value?

Reply:
This is the varplayer value used for the countdown timer to the next Wintertodt match

Reply:
Went ahead and added a comment above the enum to better clarify

Reply:
What I meant was - what is it? Seconds? MS? game ticks? A combination of both?

Reply:
It ends up being minutes/seconds on the client interface but I couldn't quite figure out how to translate the value into seconds

I have a sample of what the values look like when sent to the client in the description of the PR